### PR TITLE
Add custom.css stylesheet in login.php at isis template

### PR DESCRIPTION
**Pull request by juankadiaz**
Very usefull to customize with the same custom.css file the login.php

### CODE REVIEW

File: administrator/templates/isis/login.php [hunk 1]
```diff
@@ -24,6 +24,9 @@
 // Add Stylesheets
 $doc->addStyleSheet($this->baseurl . '/templates/' . $this->template . '/css/template' . ($this->direction == 'rtl' ? '-rtl' : '') . '.css');
 
+// Add CustomStylesheets
+$doc->addStyleSheet($this->baseurl . '/templates/' . $this->template . '/css/custom' . ($this->direction == 'rtl' ? '-rtl' : '') . '.css');
```
Comment (Fedik) [inline administrator/templates/isis/login.php]: check need to be done, before the file add ...
index.php check as example you need to

**Comment by Fedik** [comment]
how users can the patch info need here, please add some :wink:

File: administrator/templates/isis/login.php (hunk 1 above)
Comment (juankadiaz) [inline administrator/templates/isis/login.php]: ok, i will check and add. thanks for your time ;)

Comment (juankadiaz) [inline administrator/templates/isis/login.php]: fixed. thanks for your indications Fedik.

Comment (juankadiaz) [inline administrator/templates/isis/login.php]: jumm I think that this functionaility is available in the next relase:

https://github.com/joomla/joomla-cms/blob/master/administrator/templates/isis/login.php#L38-L44

**Comment by juankadiaz** [comment]
jumm I think that this functionaility is available in the next relase:

https://github.com/joomla/joomla-cms/blob/master/administrator/templates/isis/login.php#L38-L44

sorry for the lost time

**Pull request closed without merging**